libraries/src/Application/SiteApplication.php

```diff
@@ -875,10 +875,21 @@ public function setTemplate($template, $styleParams = null)
 		{
 			$this->template = new \stdClass;
 			$this->template->template = $template;
+			$this->template->inheritable = 0;
+			$this->template->parent = '';
 
 			if ($styleParams instanceof Registry)
 			{
 				$this->template->params = $styleParams;
+				if (isset($styleParams->inheritable))
```

Comment:
```suggestion

				if (isset($styleParams->inheritable))
```
Sorry, it seems with the last correction GitHub has removed the complete empty line and not the spaces within that line. But that empty line is needed. Now PHP code style failed again. Just apply again the suggestion and this time it will be fine, I promise ;-)